ScoutSuite/providers/gcp/resources/iam/bindings_separation_duties.py

```diff
@@ -0,0 +1,63 @@
+from ScoutSuite.providers.base.resources.base import Resources
+from ScoutSuite.providers.gcp.facade.base import GCPFacade
+
+
+class BindingsSeparationDuties(Resources):
+    def __init__(self, facade: GCPFacade, project_id: str):
+        super().__init__(facade)
+        self.project_id = project_id
+
+    async def fetch_all(self):
+        raw_bindings = await self.facade.cloudresourcemanager.get_member_bindings(self.project_id)
+        binding_id, binding = await self._parse_binding_separation(raw_bindings)
+        self[binding_id] = binding
+        x=1
+
+    async def _parse_binding_separation(self, raw_bindings):
+        binding_dict = {}
+        binding_dict['id'] = self.project_id
+        binding_dict["account_separation_duties"] = self.ensure_seperation_duties(raw_bindings)
+        binding_dict["kms_separation_duties"] = self.ensure_KMS_seperation_duties(raw_bindings)
+
+        return binding_dict['id'], binding_dict
+
+    def ensure_seperation_duties(self, raw_bindings):
```

Comment:
Would be good to add a description to these functions to explain what they are doing.

Reply:
Done